migrations/20190425110658-create-users.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const usersMigration = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        type: Sequelize.INTEGER
+      },
+      username: {
+        type: Sequelize.STRING,
+        unique: true,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false
+      },
+      bio: {
+        type: Sequelize.STRING,
+        allowNull: false
```

Comment:
We need to allow null for users `bio` so we will be able to register a new user. When creating a user on registration event the information we are getting from the user doesn't include `bio`.